src/components/SummarySection.jsx

```diff
@@ -0,0 +1,50 @@
+import PropTypes from "prop-types";
+import Go123Box from "./Go123Box";
+import SummaryBox from "./SummaryBox";
+
+const SummarySection = ({ className, gridClassName, summaryList }) => {
+  return (
+    <div className={className}>
+      <h2 className="pb-10 uppercase text-h3">The Summary</h2>
+      <div className={`grid ${gridClassName}`}>
+        {summaryList.map((item, i) =>
```

Comment:
for this specific case it's not as big of a concern since we're not modifying the items but it's bad React code to use the index for the key: https://react.dev/learn/rendering-lists#rules-of-keys:~:text=throughout%20its%20lifetime.-,Pitfall,-You%20might%20be